spec/ftp_spec.rb

```diff
@@ -7,14 +7,15 @@ class FtpUploader < CarrierWave::Uploader::Base
 
 describe CarrierWave::Storage::FTP do
   before do
-    CarrierWave.configure do |config|
+    FtpUploader.configure do |config|
```

Comment:
Can you explain this rename? Maybe I'm missing something but I don't see how this fits in with the rest of the PR?

Reply:
`CarrierWave.configure` is an alias for `CarrierWave::Uploader::Base.configure`. So calling `CarrierWave.configure` won't change the config values that `FtpUploader` inherited from `CarrierWave::Uploader::Base` when it was created.

An alternative to that rename would be to re-create the `FtpUploader` on each test run so that it picks up any changes that the tests have made to the config.